pandas/core/categorical.py

```diff
@@ -438,10 +492,11 @@ def _validate_categories(cls, categories, fastpath=False):
             dtype = None
             if not hasattr(categories, "dtype"):
                 categories = _convert_to_list_like(categories)
-                # on categories with NaNs, int values would be converted to float.
-                # Use "object" dtype to prevent this.
+                # On categories with NaNs, int values would be converted to
+                # float. Use "object" dtype to prevent this.
                 if isnull(categories).any():
-                    without_na = np.array([x for x in categories if notnull(x)])
+                    without_na = np.array([x for x in categories if notnull(x)
+                                           ])
```

Comment:
it would be better to put `if notnull(x)` on the same line as the `])` it the end bracket doesn't fit in width